configuration/src/main/java/io/camunda/configuration/NodeIdProvider.java

```diff
@@ -194,8 +221,30 @@ public void setTaskId(final String taskId) {
     }
   }
 
+  public static final class StaticConfig {
+    private int nodeId;
+
+    public int getNodeId() {
+      return UnifiedConfigurationHelper.validateLegacyConfiguration(
+          Cluster.PREFIX + ".node-id",
+          nodeId,
+          Integer.class,
+          UnifiedConfigurationHelper.BackwardsCompatibilityMode.SUPPORTED,
+          Set.of(Cluster.LEGACY_NODE_ID_PROPERTY));
```

Comment:
To confirm - the new node-id property = `camunda.cluster.node-id-provider.static.node-id`? 

:x: The legacy properties should include both `zeebe.broker.cluster.nodeId` and the new unified property `camunda.cluster.nodeid`